frontend/src/hooks/dashboard/useParticipantDashboard.ts

```diff
@@ -5,28 +5,54 @@ import { isNil } from '@/utils/type';
 
 import type { Dashboard, Rank } from './types';
 
+const INTERVAL_TIME = 5000;
+
 export function useParticipantDashboard() {
   const { socket } = useContext(SocketContext);
   const [ranks, setRanks] = useState<Rank[]>([]);
   const [myRank, setMyRank] = useState<Rank>({
+    rank: 0,
     email: '',
     score: 0,
     problemDict: {},
   });
   const [totalProblemCount, setTotalProblemCount] = useState(-1);
 
+  const fetchData = () => {
+    if (!isNil(socket)) {
+      socket.emit('dashboard');
+    }
```

Comment:
이건 개인적인 취향인데

```suggestion
    if (isNil(socket)) return;

    socket.emit('dashboard');
```

같이 쓰는건 어때요

Reply:
오 좋은 것 같아요, 감사합니다!